opengb/seed-core
# GRID-8338: :twisted_rightwards_arrows: Update SEED to latest release (v2.17.0)

#### Any background context you want to provide?
We would like to make use of the latest released version of SEED (v2.17.0). However, we have some customizations to outbound emails in our forked version of SEED. So, we have to carefully merge our customized version of the code with the upstream version.

#### What's this PR do?
Updates the branch containing our customized version of SEED with the changes from v2.17.0 in the upstream repository.

#### How should this be manually tested?
- Need to ensure that outbound emails still work and have the content we expect.
- Need to ensure you can still log in to SEED, upload property data, create orgs and cycles, etc.
- Monthly data upload is broken in the version of SEED that we are currently using in production (v2.15.0), so we should double-check that this new version actually fixes the bugs in that feature.

#### What are the relevant tickets?
[GRID-8338](https://opentech-cast.monday.com/boards/1256768982/pulses/3213088338)

#### Screenshots (if appropriate)
N/A
